Sources/Brave/Frontend/Settings/Features/ShieldsPrivacy/FilterLists/FilterListsView.swift

```diff
@@ -78,8 +77,23 @@ struct FilterListsView: View {
       } header: {
         Text(Strings.customFilterLists)
       }
+      .listRowBackground(Color(.secondaryBraveGroupedBackground))
+      .toggleStyle(SwitchToggleStyle(tint: .accentColor))
       
       Section {
+        #if DEBUG
+        let allToggled = Binding<Bool> {
+          self.filterListStorage.filterLists.allSatisfy({ $0.isEnabled })
+        } set: { allToggled in
+          for index in 0..<filterListStorage.filterLists.count {
+            filterListStorage.filterLists[index].isEnabled = allToggled
+          }
+        }
+
+        Toggle("All", isOn: allToggled)
+          .toggleStyle(SwitchToggleStyle(tint: .accentColor))
+        #endif
+        
```

Comment:
Some debugging functionality so I can test while compiling/uncomplying all the engines.